modules/openapi-generator/src/main/resources/Java/typeInfoAnnotation.mustache

```diff
@@ -1,6 +1,9 @@
 {{#jackson}}
 
-@com.fasterxml.jackson.annotation.JsonIgnoreProperties(value = "{{{discriminator.propertyBaseName}}}", allowSetters = true)
+@com.fasterxml.jackson.annotation.JsonIgnoreProperties(
```

Comment:
I would prefer to set up a proper include using model. To do you'll have to add an importMapping for `JsonIgnoreProperties` in `AbstractJavaCodegen`. Let me know if you need help to do so.